Source/Core/VideoBackends/Vulkan/VKTexture.cpp

```diff
@@ -290,7 +290,7 @@ void VKTexture::Load(u32 level, u32 width, u32 height, u32 row_length, const u8*
   if (upload_size <= STAGING_TEXTURE_UPLOAD_THRESHOLD &&
       upload_size <= MAXIMUM_TEXTURE_UPLOAD_BUFFER_SIZE)
   {
-    StreamBuffer* stream_buffer = TextureCache::GetInstance()->GetTextureUploadBuffer();
+    StreamBuffer* stream_buffer = g_object_cache->GetTextureUploadBuffer();
```

Comment:
Just curious, why this was necessary?

Reply:
Texture cache is initialized after renderer, and the base renderer class uploads the imgui font atlas, so this buffer must exist already.